CHANGELOG.md

```diff
@@ -1,4 +1,48 @@
-# Changelog
+# CHANGELOG
+
+## 1.25.0 (2022-04-11)
+
+MLflow 1.25.0 includes several major features and improvements:
+
+Features:
+
+- [Artifacts] Introduce a new `mlflow.artifacts.download_artifacts()` API that mirrors functionality of CLI `download_artifacts` (#5585, @dbczumar)
+- [Models] Add support for saving artifact if destination is an empty directory. (#5572, @akshaya-a)
+- [Models] Introduce support for `diviner` as a flavor. (#5553, @BenWilson2)
+- [Models] Add `mlflow_version` to logged model metadata. (#5515, @r3stl355)
+- [Models] Add `pyfunc.get_model_dependencies()` API to retrieve reproducible environment configuration for pyfunc flavor. (#5503, @WeichenXu123)
+- [Models] Add input example and signature autologging for `tensorflow.keras`. (#5461, @bali0019)
+- [Models] Add `code_paths` argument to all model flavors to support logging custom module code for external-dependencies for a model. (#5448, @stevenchen-db)
+- [Models / Projects] Introduce `--env-manager` configuration to prepare for support for pyenv & virtualenv. Deprecate `--no-conda` (#5567, @harupy)
+- [Scoring] Introduce `--env-manager` configuration for selection of either local or conda environment for serving. (#5487, @WeichenXu123)
+- [Scoring] Add support for `numpy.ndarray` type inputs to `tensorflow` predict function (#5545, @WeichenXu123)
+- [Scoring] Add support for Docker image deployment to Sagemaker Serverless (#5610, @matthewmayo)
+- [Tracking] Introduce a new fluent API `mlflow.last_active_run()` that will return `mlflow.active_run()` when called. (#5584, @MarkYHZhang)
```

Comment:
Can we list Tracking updates first, since Tracking is the most widely-used MLflow component?

Reply:
Tracking -> Models -> Scoring -> UI -> Artifacts